.kitchen.yml

```diff
@@ -11,6 +11,9 @@ platforms:
   - name: ubuntu-12.04
   - name: debian-6.0.8
   - name: centos-6.4
+    run_list:
+      - recipe[yum-epel::default]
+      - recipe[yum-remi::default]
```

Comment:
Hi,

It's preferable to not include them IMHO because it's up to the user to choose which version he wants to install. Let's say the user has a private repo with a specific version of Redis inside, it may overlap the wished version. In addition, user may not want to have those repository installed at all.

Those repo are not mandatory but recommended to get the latest Redis version. That's why I think it's better to keep it that way.